packages/manager/modules/overthebox/src/overthebox/autoconfigure/dns/autoconfigure-dns.controller.js

```diff
@@ -0,0 +1,226 @@
+    }
+
+    const params = {
+      configuration: {
+        server: this.nameserver.server,
+      },
+      priority: this.nameserver.priority,
+    };
+
+    return this.$http
+      .post(
+        `/overTheBox/${this.serviceName}/configuration/dns/nameserver`,
+        params,
+      )
+      .then((response) => {
+        // Display success message
+        this.successMessage = this.$translate.instant(
+          'overTheBox_autoconfigure_dns_server_create_succeed',
+        );
+
+        // Reload nameserver table
+        this.loadNameserver();
+        this.showAddServer = false;
+        this.displayAddServerButton = true;
+        return response;
+      })
+      .catch((error) => {
+        // Display error message
+        this.errorMessage = this.$translate.instant(
+          'overTheBox_autoconfigure_create_failed',
+          { errorMessage: error.data.message },
+        );
+      });
+  }
+
+  cancelCreateNameServer() {
+    this.showAddServer = false;
+    this.displayAddServerButton = true;
+  }
+
+  removeNameserver(row) {
+    this.errorMessage = '';
+    this.successMessage = '';
+    return this.$http
+      .delete(
+        `/overTheBox/${this.serviceName}/configuration/dns/nameserver/${row.id}`,
+      )
+      .then((response) => {
+        // Display success message
+        this.successMessage = this.$translate.instant(
+          'overTheBox_autoconfigure_dns_server_remove_succeed',
+        );
+
+        // Reload nameserver table
+        this.loadNameserver();
+        return response;
+      })
+      .catch((error) => {
+        // Display error message
+        this.errorMessage = this.$translate.instant(
+          'overTheBox_autoconfigure_remove_failed',
+          { errorMessage: error.data.message },
+        );
+      });
+  }
+
+  addLocalDomain() {
+    this.localDomain.hostname = '';
+    this.localDomain.ip = '';
+    this.localDomain.priority = '';
+    this.showAddLocalDomain = true;
+    this.displayAddLocalDomainButton = false;
+  }
+
+  createLocalDomain() {
+    this.errorMessage = '';
+    this.successMessage = '';
+    if (
+      !this.localDomain.hostname &&
+      !this.localDomain.ip &&
+      !this.localDomain.priority
+    ) {
+      return this.$q.resolve(null);
+    }
+
+    const params = {
+      configuration: {
+        hostname: this.localDomain.hostname,
+        ip: this.localDomain.ip,
+      },
+      priority: this.localDomain.priority,
+    };
+
+    return this.$http
+      .post(
+        `/overTheBox/${this.serviceName}/configuration/dns/localDomain`,
+        params,
+      )
+      .then((response) => {
+        // Display success message
+        this.successMessage = this.$translate.instant(
+          'overTheBox_autoconfigure_dns_local_domain_create_succeed',
+        );
+
+        // Reload local domain table
+        this.loadLocalDomain();
+        this.showAddLocalDomain = false;
+        this.displayAddLocalDomainButton = true;
+        return response;
```

Comment:
`response` seems not used
```suggestion
        this.showAddLocalDomain = false;
        this.displayAddLocalDomainButton = true;
        return this.loadLocalDomain();
```